### DISCUSSION TITLE
Ctrl++

### DISCUSSION THREAD

**Discussion by raccoon666666** [shuax/MouseInc.Settings#226] [General]
想设置一个"Ctrl+加号"的手势, 但是我写成Ctrl++发现不好使呀, 有什么方法可以支持按键加号吗??????

**Comment by raccoon666666** [comment]
流汗了😅

**Comment by shuax** [comment]
使用plus代替+。
https://docs.shuax.com/MouseInc/#/action#sendkeys

**Reply by raccoon666666** [reply]
好使了, 谢谢
